src/Core/Feature.js

```diff
@@ -223,6 +223,7 @@ function push3DValues(value0, value1, value2 = 0) {
  * @property {number[]} normals - All the normals of the Feature.
  * @property {number} size - the number of values of the array that should be associated with a coordinates.
  * The size is 3 with altitude and 2 without altitude.
+ * @property {boolean} hasRawElevationData - indicates if the raw data,from original source, has elevation.
```

Comment:
I would add `(i.e. is 3D)` at the end of the comment to be even clearer

Reply:
yes why not but I would still add explicitly something like "is 3D" or "has a third coordinate"

Reply:
ok 👍🏽